distribution/kpkginstall/runtest.sh

```diff
@@ -38,14 +44,29 @@ function set_package_name()
       PACKAGE_NAME=kernel
   fi
 
+  # Append "-debug" if we were asked to install the debug kernel.
+  if [[ "${KPKG_INSTALL_DEBUG:-}" == "yes" ]]; then
+    echo "Debug kernel was requested -- appending -debug to package name"
+    PACKAGE_NAME=${PACKAGE_NAME}-debug
+  fi
+
+  # Write the PACKAGE_NAME to a file in /tmp so we have it after reboot.
+  echo -n "${PACKAGE_NAME}" | tee -a /tmp/kpkginstall/KPKG_PACKAGE_NAME
```

Comment:
I wonder if `/tmp/kpkginstall/KPKG_PACKAGE_NAME` and co. should be stored in "constants"...